Inline the `_fetchDocument` helper function in `getDocument`

Snuffleupagus opened the pull request:
This function has been modified a number of times over the years, and at this point it's small/simple enough that we can just inline the code instead.

Snuffleupagus committed:
Inline the `_fetchDocument` helper function in `getDocument`

This function has been modified a number of times over the years, and at this point it's small/simple enough that we can just inline the code instead.

nicolo-ribaudo commented on src/display/api.js:
```diff
@@ -398,9 +398,16 @@ function getDocument(src) {
+        docParams,
+        data ? [data.buffer] : null
+      );
+      const networkStreamPromise = new Promise(resolve => {
```
`resolve` inside this Promise constructor is always called synchronously -- is it actually needed to have it as a promise?

Snuffleupagus replied:
As with many other things that were once necessary, other changes have removed the need for this to be async :-)

Snuffleupagus committed:
Initialize the `networkStream` synchronously in `getDocument`

This is fairly old code, and at some point the need for this to be asynchronous disappeared.

timvandermeij commented:
/botio-linux preview

timvandermeij commented:
/botio test
